scripts/qiita

```diff
@@ -425,18 +424,5 @@ def start(port):
     tornado.ioloop.IOLoop.instance().start()
 
 
-# #############################################################################
-# WARE COMMANDS
-# #############################################################################
-
-@ware.command()
-@click.argument('sl_out_dir', required=True,
-                type=click.Path(resolve_path=True, readable=True, exists=True,
-                                file_okay=False))
-def generate_demux(sl_out_dir):
```

Comment:
generate demux is part of the plugin. I'm unsure if we want to add CLI to the plugins... I would suggest adding an issue to see what we can do. Note that this functionality is available anyways through ipython, by importing the generate_demux_file func from the plugin